application/models/User.php

```diff
@@ -292,4 +292,51 @@ public function relations()
 			'permissions' => array(self::HAS_MANY, 'Permission', 'uid')
 		);
 	}
+
+    /**
+    * Returns Users from string id list
+    *
+    * @access public
+    * @return array User list
+    */
+    public static function getArrayUsers($stringUsersList, $outputformat='fullinfoarray')
```

Comment:
Please add documentation for all function arguments.
Example:
`@param string $stringUsersList Comma-separated list with user ids`